tests/capi/unittest_util.c

```diff
@@ -57,6 +58,52 @@ guint get_available_port (void)
   return port;
 }
 
+/**
+ * @brief Prepare conf file for unittest, update available port number.
+ */
+gchar *
+prepare_test_config (const gchar * conf_base, const gchar * port_name, const guint port)
+{
+  const gchar *tmp_dir = g_get_tmp_dir ();
+  gchar *tmp_conf = NULL, *res_conf = NULL;
+  gchar *contents = NULL;
+  gchar *cmd = NULL;
+  gint status;
+
+  if (tmp_dir == NULL) {
+    return NULL;
+  }
+
+  tmp_conf = g_build_filename (tmp_dir, "ml_conf_tmp_XXXXXX.conf", NULL);
+  status = g_mkstemp (tmp_conf);
+  if (status < 0) {
+    goto error;
+  }
+
+  g_close (status, NULL);
+
+  if (!g_file_get_contents (conf_base, &contents, NULL, NULL) ||
+      !g_file_set_contents (tmp_conf, contents, -1, NULL)) {
+    goto error;
+  }
+
+  cmd = g_strdup_printf ("sed -i '/__AVAIL_PORT__/c\\\"%s\" : \"%u\",' %s",
```

Comment:
How about ?
sed 's/__AVAIL_PORT/port_number/g' conf_base

Reply:
updated, please review the changes.